nevergrad/optimization/base.py

```diff
@@ -411,7 +411,7 @@ def optimize(self, objective_function: Callable[..., float],
                 sleeper.sleep()
             # # # # # Start new jobs # # # # #
             if not batch_mode or not self._running_jobs:
-                new_sugg = min(remaining_budget, self.num_workers - len(self._running_jobs))
```

Comment:
You could also use `np.clip`, but in this case you may need to cast to int afterwards, dunno if it is worth it

Reply:
Yes true but I think that:
`np.clip(min(remaining_budget, self.num_workers - len(self._running_jobs)), a_min=0, a_max=None)`
if not much clearer to justify using numpy instead of the builtins

Reply:
`np.clip(self.num_workers - len(self._running_jobs)), a_min=0, a_max=remaining_budget)` seems simpler though :p